src/less/skeleton-loader/skeleton-loader.less

```diff
@@ -0,0 +1,235 @@
+@import "../variables/variables.less";
+
+@skeleton-text-border-radius: 3px;
+@skeleton-text-height-small: 16px;
+@skeleton-text-height-large: 24px;
+
+:root {
+    --skeleton-border-radius: 0;
+    --skeleton-height: 48px;
+    --skeleton-tile-width-max: 224px;
+    --skeleton-tile-width-min: 121px;
+    --skeleton-width: 100%;
+}
+
+.skeleton {
+    background: var(--color-loading-fill);
+    border-radius: var(--skeleton-border-radius);
+    cursor: progress;
+    height: var(--skeleton-height);
+    min-height: 12px;
+    width: var(--skeleton-width);
+}
+
+.skeleton-elevated {
+    background: var(--color-loading-elevated-fill);
+}
+
+.skeleton-ai-gradient-purple-subtle {
+    background: var(--color-ai-gradient-purple-subtle);
+}
+
+.skeleton-ai-gradient-green-subtle {
+    background: var(--color-ai-gradient-green-subtle);
+}
+
+.skeleton-ai-gradient-blue-subtle {
+    background: var(--color-ai-gradient-blue-subtle);
+}
+
+.skeleton--avatar {
+    border-radius: 50%;
+    height: var(--skeleton-height);
+    width: var(--skeleton-height);
+}
+
+.skeleton--btn {
+    border-radius: 20px;
+    height: 40px;
+}
+
+.skeleton--btn-small {
+    border-radius: 16px;
+    height: 32px;
+}
+
+.skeleton--btn-large {
+    border-radius: 24px;
+    height: 48px;
+}
+
+.skeleton--text-small {
+    border-radius: @skeleton-text-border-radius;
+    height: @skeleton-text-height-small;
+}
+
+.skeleton--text-small::after {
+    height: @skeleton-text-height-small;
+    margin-top: calc(@skeleton-text-height-small + var(--spacing-100));
+}
+
+.skeleton--text-large {
+    border-radius: @skeleton-text-border-radius;
+    height: @skeleton-text-height-large;
+}
+
+.skeleton--text-large::after {
+    height: @skeleton-text-height-large;
+    margin-top: calc(@skeleton-text-height-large + var(--spacing-100));
+}
+
+.skeleton--text-multiline {
+    position: relative;
+    width: calc(100% - var(--spacing-300));
+}
+
+.skeleton--text-multiline::after {
+    background: inherit;
+    content: "";
+    position: absolute;
+    width: calc(100% - var(--spacing-700));
+}
+
+.skeleton--text-small.skeleton--text-multiline {
+    margin-bottom: var(--spacing-300);
+}
+
+.skeleton--text-large.skeleton--text-multiline {
+    margin-bottom: var(--spacing-400);
+}
+
+.skeleton--form {
+    border-radius: 8px;
+    height: 48px;
+}
+
+// TODO: Use aspect-ratio : 1 / 1 to set as square once browser support for safari > 15
+.skeleton--image {
+    border-radius: 8px;
+    height: 100%;
+    max-width: initial;
+}
+
+.skeleton-image-container {
+    container-name: skeleton-image;
+    container-type: inline-size;
+
+    .skeleton.skeleton--image {
```

Comment:
Lets move this to not be nested, so like 
`.skeleton-image-container .skeleton.skeleton--image {`
That way when we want to look it up, its easier to find.